src/common/components/clusterWizard/networkingSteps/NetworkTypeControlGroup.tsx

```diff
@@ -13,46 +13,52 @@ export const NetworkTypeControlGroup: React.FC<NetworkTypeControlGroupProps> = (
   const GROUP_NAME = 'networkType';
 
   return (
-    <Split hasGutter>
-      <SplitItem>
-        <Tooltip
-          hidden={!isIPv6}
-          content={'Software-Defined Networking (SDN) can be selected only when IPv4 is detected.'}
-        >
+    <FormGroup fieldId={GROUP_NAME} label="Network type">
+      <Split hasGutter>
+        <SplitItem>
+          <Tooltip
+            hidden={!isIPv6}
+            content={
+              'Software-Defined Networking (SDN) can be selected only when IPv4 is detected.'
+            }
+          >
+            <RadioField
+              id={GROUP_NAME}
+              name={GROUP_NAME}
+              isDisabled={isIPv6}
+              value={'OpenShiftSDN'}
+              label={
+                <>
+                  Software-Defined Networking (SDN){' '}
+                  <PopoverIcon
+                    variant={'plain'}
+                    bodyContent={'The classic bullet-proof networking type'}
+                  />
+                </>
+              }
+            />
+          </Tooltip>
+        </SplitItem>
+        <SplitItem />
+        <SplitItem>
           <RadioField
+            id={GROUP_NAME}
             name={GROUP_NAME}
-            isDisabled={isIPv6}
-            value={'OpenShiftSDN'}
+            value={'OVNKubernetes'}
             label={
               <>
-                Software-Defined Networking{' '}
+                Open Virtual Networking (OVN){' '}
                 <PopoverIcon
                   variant={'plain'}
-                  bodyContent={'The classic bullet-proof networking type'}
+                  bodyContent={
+                    "The next generation networking type, select this when you're using new features and telco features"
```

Comment:
I would add dot `.` at the end (sentence).
I am not sure about the first part before comma - I would start a new sentence with the `Select` word.
A copy-writer's opinion would be nice here.

Reply:
These are the texts I received from @nirfarkas (https://issues.redhat.com/browse/MGMT-6769), this task only adds the acronyms in between parenthesis.